### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by urbaman** [mattieserver/netbox-topology-views#257] [Q&A]
### NetBox version

v3.4.3

### Topology Views version

3.2.1

### Steps to Reproduce

source /opt/netbox/venv/bin/activate
cd /opt/netbox/netbox

### Expected Behavior

Installing the package

### Observed Behavior

Collecting netbox-topology-views
  Using cached netbox_topology_views-3.2.1-py3-none-any.whl (360 kB)
Installing collected packages: netbox-topology-views
ERROR: Could not install packages due to an OSError: [Errno 13] Permission denied: '/opt/netbox-3.4.3/venv/lib/python3.10/site-packages/netbox_topology_views'
Check the permissions.

**Comment by dreng** [comment]
The error message does not indicate a bug, but a permission problem in your environment. I have therefore converted the bug report into a discussion.

A little hint: "Error" is not a meaningful title. This reduces the chances that someone will read your post.

**Reply by mattieserver** [reply]
You might need to use 'sudo' before the install command. (or run as the user that owns the venv)
